antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -325,33 +338,74 @@ def get_study_information(
         Returns: study metadata
 
         """
-        return get_study_information(
+        self._safe_generation(study)
+
+        return super().get_study_information(
             study,
-            study.snapshot.path if study.snapshot is not None else None,
-            self.patch_service,
-            self.study_factory,
-            logger,
             summary,
         )
 
-    def get_study(self, study_id: str) -> FileStudy:
+    def get(
+        self,
+        metadata: VariantStudy,
+        url: str = "",
+        depth: int = 3,
+        formatted: bool = True,
+    ) -> JSON:
         """
-        Get study
+        Entry point to fetch data inside study.
         Args:
-            study_id: study id
-        Returns: Study
+            metadata: study
+            url: path data inside study to reach
+            depth: tree depth to reach after reach data path
+            formatted: indicate if raw files must be parsed and formatted
+
+        Returns: study data formatted in json
+
         """
-        raise NotImplementedError()
+        self._safe_generation(metadata)
+
+        return super().get(
+            metadata=metadata,
+            url=url,
+            depth=depth,
+            formatted=formatted,
+        )
 
-    def import_output(self, study: Study, output: Union[bytes, Path]) -> None:
+    def edit_study(
+        self, metadata: VariantStudy, url: str, new: SUB_JSON
+    ) -> SUB_JSON:
         """
-        Import an output
+        Replace data on disk with new
         Args:
-            study: the study
-            output: Path of the output or raw data
-        Returns: None
+            metadata: study
+            url: data path to reach
+            new: new data to replace
+
+        Returns: new data replaced
+
         """
-        raise NotImplementedError()
+        self._safe_generation(metadata)
+        return super().edit_study(metadata=metadata, url=url, new=new)
```

Comment:
set this as NotImplemented.
The edition here will create new commands from the input. (#501)